render/id.go

```diff
@@ -14,6 +24,11 @@ func MakeEndpointID(hostID, addr, port string) string {
 	return makeID("endpoint", hostID, addr, port)
 }
 
+// MakePseudoEndpointID makes a pseudo endpoint node ID for rendered nodes.
+func MakePseudoEndpointID(hostID, addr, port string) string {
+	return makeID("pseudo-endpoint", hostID, addr, port)
+}
```

Comment:
Seems an odd differentiation between `pseudo` nodes and `pseudo-endpoint` nodes.